lib/public_api_v1.ts

```diff
@@ -118,23 +118,23 @@ const appInsightsClient = new ApplicationInsights.TelemetryClient();
 // Setup handlers
 
 const debugHandler = GetDebug(organizationModel);
-app.get("/api/v1/debug", debugHandler);
-app.post("/api/v1/debug", debugHandler);
+app.get("/debug", debugHandler);
```

Comment:
you should also update [function.json](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/PublicApiV1/function.json)

Reply:
anyway, I'm not much in agreement about this, since it prevents having another HTTP Function handling the v2 API within this application (e.g. a `public_api_v2.ts`) - I see more downsides than upsides (what is the upside anyway?)

Reply:
upside is you can plug multiple versions of the API in the developer portal to test it inside the UI (but I'm not particularly biased). if we want to maintain the prefix maybe we should do this for the swagger.json endpoint as well ?

    app.get("/swagger.json", GetOpenapi(publicApiV1Specs));